tests/test_copyright.py

```diff
@@ -48,11 +50,18 @@ def copyright_is_valid(fname):
     if "Google Inc" in lines[1]:
         apache_line += 1
 
-    # The Copyright includes the current year
+    # The Copyright includes valid years
     current_year = date.today().year
-    if str(current_year) not in lines[0]:
+    re_years = re.compile(r"[-\s](?P<year>\d+)")
+    years = map(int, re_years.findall(lines[0]))
```

Comment:
You don't get anything by compiling the regexp.  It's more efficient and compact to just run `re.findall` on the line:

    years = map(int, re.findall(r"[-\s](\d+)", lines[0]))